DOC-2363: Notification width was not constrained to the width of the editor. (#3268)
tinymce/tinymce-docs: modules/ROOT/pages/7.1-release-notes.adoc

{productname} {release-version} also includes the following bug fixes:

=== Notification width was not constrained to the width of the editor.
// #TINY-10886

> Previously, opening a notification or when one is display in the editor would cause it to exceed the editor's fixed width if it contained a large amount of text.
> 
> As a consequence, the notification became wider than the editor.
> 
> In {productname} {release-version}, this issue is resolved by ensuring that notifications do not exceed the width of the editor.

Reviewer: ```suggestion
Previously, when opening a notification containing a large amount of text or when one was displayed by the editor, the notification was not constrained to the width of the editor.

As a consequence, the notification popup would exceed the fixed width of the editor window.

In {productname} {release-version}, this issue is resolved by ensuring that notifications do not exceed the width of the editor.
```
